dist/common/scripts/scylla_io_setup

```diff
@@ -176,6 +176,9 @@ def run_iotune():
                 sys.exit(1)
 
 if __name__ == "__main__":
+    if not is_nonroot() and os.getuid() > 0:
+        print('Requires root permission.')
+        sys.exit(1)
```

Comment:
Why does it fail without root? The files should be written to the user directory, not /etc.

Reply:
if it's NOT nonroot mode, then the script will write to /etc, requires root privilege.

Reply:
We actually already do same in scylla_setup:
 https://github.com/scylladb/scylla/blob/master/dist/common/scripts/scylla_setup#L184
So #8915 only occurs when we run scylla_io_setup directly, but it's still better to fix.

Reply:
@syuu1228 Perhaps as a follow-up, we ought to rename `is_nonroot()` to `is_user()` to avoid the confusion.